### ISSUE TITLE
zoom always displays 100% on hamburger menu

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-laptop#14045]
### Description
When you zoom either via shortcut or hamburger menu, the value displayed on the hamburger menu should reflect your zoom level. With 0.22.702 it always says 100% even after zooming.

### Steps to Reproduce

  1. Open a site.
  2. Zoom in or out (via shortcut or hamburger menu.
  3. 

**Actual result:**
Value displayed on hamburger menu is not indicative of your zoom level, it just displays 100%.

**Expected result:**
Zoom displayed on hamburger menu should reflect your zoom level.

**Reproduces how often:**
Easily

### Brave Version

**about:brave info:**
0.22.702

**Reproducible on current live release:**
no, does not reproduce on 0.22.702

### Additional Information
Reproduced on macOS by @kjozwiak and Linux by @btlechowski

**Comment by petemill** [comment]
Should be fixed in next build via 87337338934a92743724b585cdaf0b31bab10e0b

**Comment by btlechowski** [comment]
Still reproducible.
```
Brave: 0.22.703 
V8: 6.6.346.26 
rev: 903b8d0d3703b5fdc5a719294403e7460d9b806d 
Muon: 6.0.8 
OS Release: 4.13.0-21-generic 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: Linux 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 66.0.3359.139
```

**Comment by petemill** [comment]
Should be fixed in next build via f4c5b68

**Comment by LaurenWags** [comment]
Verified with macOS 10.12.6 using
* 0.22.706 e11b027
* muon 6.0.9
* libchromiumcontent 66.0.3359.139
